config/environment.js

```diff
@@ -37,6 +37,9 @@ module.exports = function(environment) {
 
     web3ProviderUrl: 'https://rinkeby.infura.io/v3/d4f788b7a6584f7db2fc3c268d4d09e9',
 
+    githubConnectUrl: 'https://hal8000.chat.kosmos.org:8082/kredits/signup/connect/github',
```

Comment:
Why is this using a port number? This should actually fail, as the HTTPS nginx sites are not listening on those ports.

https://gitea.kosmos.org/kosmos/chef/src/branch/master/site-cookbooks/kosmos-hubot/templates/default/nginx_conf_hubot.erb#L10

Reply:
Oh, yes. I didn't know the actual URL, so I searched the chef repo and found this: https://gitea.kosmos.org/kosmos/chef/src/branch/master/site-cookbooks/kosmos-hubot/attributes/default.rb#L6-L7

It's fixed now.